server/master/src/main/java/org/apache/accumulo/master/Master.java

```diff
@@ -1300,7 +1300,7 @@ public void run() throws IOException, InterruptedException, KeeperException {
     zReaderWriter.getChildren(zroot + Constants.ZRECOVERY, new Watcher() {
       @Override
       public void process(WatchedEvent event) {
-        nextEvent.event("Noticed recovery changes", event.getType());
+        nextEvent.event("Noticed recovery changes %s", event.getType());
```

Comment:
Is this printf style parameterization? I guess I assumed that the parameter was unrelated to the description here.

Reply:
The event method takes a string but eventually prints using String.format()